tests/kv_test/train_all.sh

```diff
@@ -1,59 +1,50 @@
 set -e
 echo "Training with compiler flags ${CXX} ${CXXFLAGS} ${CXXBACKFLAGS}"
 
+ROOT=$1
+STORAGE=$2
+DATA_PATH="${HOME}/data"
+STORAGE_PATH="${ROOT}/${STORAGE}"
+BINARY_PATH="${ROOT}/binary"
+
+echo ${STORAGE_PATH}
+mkdir -p ${ROOT}
+mkdir -p ${STORAGE_PATH}
+mkdir -p ${BINARY_PATH}
+
 DATASET_NAMES=(
-  "books_800M_uint64"
-  "fb_200M_uint64"
-  "lognormal_200M_uint64"
-  "normal_200M_uint64"
-  "osm_cellids_800M_uint64"
-  "uniform_dense_200M_uint64"
-  "uniform_sparse_200M_uint64"
-  "wiki_ts_200M_uint64"
- )
-# RMI_CONFIGS=( # configurations obtained from rmi optimizer
-#   "linear_spline,linear      32768" # books_800M_uint64
-#   "robust_linear,linear      32768" # fb_200M_uint64
-#   "robust_linear,linear 32768" # lognormal_200M_uint64
-#   "robust_linear,linear 32768" # normal_200M_uint64
-#   "linear,linear             32768" # osm_cellids_800M_uint64
-#   "linear_spline,linear      32768" # uniform_dense_200M_uint64
-#   "linear_spline,linear      32768" # uniform_sparse_200M_uint64
-#   "linear,linear             32768" # wiki_ts_200M_uint64
-# )
+ "wiki_ts_200M_uint64"
+ "osm_cellids_800M_uint64"
+ "fb_200M_uint64"
+ "books_800M_uint64"
+ "gmm_k100_800M_uint64"
+)
+
+#RMI_CONFIGS=( # configurations obtained from rmi optimizer
+# "linear_spline,linear  16777216"
+# "cubic,linear  16777216"
+# "robust_linear,linear  16777216"
+# "linear_spline,linear  16777216"
+# "linear_spline,linear 1048576"
+#)
+
 RMI_CONFIGS=( # configurations obtained from rmi optimizer
-  "linear_spline,linear   16777216" # books_800M_uint64
-  "robust_linear,linear   16777216" # fb_200M_uint64
-  "robust_linear,linear 16777216" # lognormal_200M_uint64
-  "robust_linear,linear 16777216" # normal_200M_uint64
-  "cubic,linear           16777216" # osm_cellids_800M_uint64
-  "radix,linear_spline     8388608" # uniform_dense_200M_uint64
-  "linear_spline,linear   16777216" # uniform_sparse_200M_uint64
-  "linear_spline,linear   16777216" # wiki_ts_200M_uint64
+ "linear_spline,linear  128"
+ "robust_linear,linear  128"
+ "robust_linear,linear  128"
+ "linear_spline,linear  128"
+ "radix,linear 128"
```

Comment:
This is the least accurate RMI, right? I believe we end up with the most accurate RMI.